src/bundler/deno_bundler.ts

```diff
@@ -0,0 +1,29 @@
+import { BundleError } from "../errors.ts";
+
+export type DenoBundleOptions = {
```

Comment:
Do we need or want to export these bundler option types?

Reply:
I exported this type mainly for future proofing but I don't think we need to export them since this is internal and any future work will be able to export as required

Reply:
I've found that, in our other typescript libraries, things that get `export`ed tend to decay over time. Maintainers don't want to remove or change the types, because they are afraid they may break something. I've seen this particularly in the node-slack-sdk project.

I say: if it's not critical for external developers, and not used outside of the file, I don't think we should export it.